### DISCUSSION TITLE
como extrair o jid das mensagens quoteds.

### DISCUSSION THREAD

**Discussion by isaias-silva** [WhiskeySockets/Baileys#237] [General]
como extrair ou encontrar um jid ou telefone do atributo quoted da mensagem, ao "marcar" e responder uma mensagem é possivel acessar o texto da mensagem respondida no atributo quoted do objeto Imessage, gostaria de saber se é possivel extrair o numero ou jid dessa mensagem.

**Comment by okfull** [comment]
`message?.extendedTextMessage?.contextInfo?.mentionedJid`

**Comment by DroidzinDev** [comment]
É simples mano

`// Obter participant da quoted

const jidQuoted = info.message[Object.keys(info.message)[0]]?.contextInfo?.participant // se quiser a jid do chat na quoted, coloque remoteJid
`

depois me fala se deu certo
